src/SFA.DAS.EmployerApprenticeshipsService.Application/Commands/CreateAccount/CreateAccountCommandHandler.cs

```diff
@@ -66,6 +68,8 @@ public CreateAccountCommandHandler(IAccountRepository accountRepository, IUserRe
             {
                 AccountId = accountId
             });
+
+            await _mediator.SendAsync(new CreateAccountEventCommand { HashedAccountId = hashedAccountId, Event = "AccountCreated" });
```

Comment:
Its possible that we are guilty of doing this elsewhere but IAsynNotification should be the approach...